myia/type/infer.py

```diff
@@ -0,0 +1,141 @@
+"""Type inference."""
```

Comment:
Remove the unification code that's in this file.

Reply:
Should be done.